Managers/PerformanceMan.cpp

```diff
@@ -0,0 +1,264 @@
+#include "PerformanceMan.h"
+#include "MovableMan.h"
+#include "FrameMan.h"
+#include "AudioMan.h"
+#include "Timer.h"
+
+#include "GUI.h"
+#include "AllegroBitmap.h"
+
+namespace RTE {
+
+	const std::string PerformanceMan::c_ClassName = "PerformanceMan";
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void PerformanceMan::Clear() {
+		m_ShowPerfStats = false;
+		m_AdvancedPerfStats = true;
+		m_CurrentPing = 0;
+		m_FrameTimer = 0;
+		m_MSPFs.clear();
+		m_MSPFAverage = 0;
+		m_SimSpeed = 1.0;
+	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	int PerformanceMan::Create() {
+		m_FrameTimer = new Timer();
+		m_Sample = 0;
+
+		for (unsigned short c = 0; c < PERF_COUNT; ++c) {
+			for (unsigned short i = 0; i < c_MaxSamples; ++i) {
+				m_PerfData[c][i] = 0;
+				m_PerfPercentages[c][i] = 0;
+			}
+			m_PerfMeasureStart[c] = 0;
+			m_PerfMeasureStop[c] = 0;
+		}
+		// Set up performance counter's names
+		m_PerfCounterNames[PERF_SIM_TOTAL] = "Total";
+		m_PerfCounterNames[PERF_ACTORS_PASS1] = "Act Travel";
+		m_PerfCounterNames[PERF_PARTICLES_PASS1] = "Prt Travel";
+		m_PerfCounterNames[PERF_ACTORS_PASS2] = "Act Update";
+		m_PerfCounterNames[PERF_PARTICLES_PASS2] = "Prt Update";
+		m_PerfCounterNames[PERF_ACTORS_AI] = "Act AI";
+		m_PerfCounterNames[PERF_ACTIVITY] = "Activity";
+
+		return 0;
+	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void PerformanceMan::Destroy() {
+		delete m_FrameTimer;
+		Clear();
+	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void PerformanceMan::ResetFrameTimer() { m_FrameTimer->Reset(); }
```

Comment:
These are here otherwise I have to include `Timer.h` in the header.

Reply:
Sorry, to clarify. I mean the method looking like
```
void PerformanceMan::ResetFrameTimer() { m_FrameTimer->Reset(); }
```
instead of
```
void PerformanceMan::ResetFrameTimer() {
	m_FrameTimer->Reset();
}
```

Reply:
I don't see how this is different from one liners in headers.
Plus we have a bunch of these in the System folder from the previous PR.

Reply:
Yeah, feels a little weird to me. Mehh, leave it be, I apparently didn't notice the ones in System before so it's a little late for me to start caring about them haha.